src/msg_recorder/record_file.cc

```diff
@@ -325,7 +326,7 @@ bool Symlink(const std::filesystem::path& to, const std::filesystem::path& from)
     std::error_code ec{};
     fs::create_symlink(to, from, ec);
     const bool has_error = ec.operator bool();
-    LOG_IF(WARNING, has_error) << __func__ << ": Failed to create symlink to " << to << "with error \"" << ec.message()
+    LOG_IF(WARNING, has_error) << __func__ << ": Failed to create symlink to " << to << " with error \"" << ec.message()
```

Comment:
`"Failed to symlink " << from << "->" << to << " with error \"" << ec.message() << "\"."`

Reply:
(pending issues)